src/lib/layer-manager.js

```diff
@@ -115,9 +115,11 @@ export default class LayerManager {
     assert(this.context.viewport, 'LayerManager.drawLayers: viewport not set');
 
     const {uniforms} = this.context;
+    let drawCount = 0;
```

Comment:
minor but shall we use layerIndex or layerZIndex instead? 
drawCount sounds like the accumulated number of rendering calls.